### PR TITLE
Add check for NGINX starting up and common time vars

### PR DESCRIPTION
**Pull request by MSevey**
This PR adds the check that the NGINX docker container is up and not restarting.  It has a 15min timeout.

I added a note saying that the check is temporary, needed for next week's deploy. But it should be an instant pass in normal cases so there is no real harm in leaving it in.

### CODE REVIEW

File: playbooks/tasks/portals-check-nginx.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Wait until nginx is no longer restarting
+- name: Check for Restarting in docker ps
+  shell: docker ps | grep nginx
+  register: docker_nginx
+  until: docker_nginx.stdout.find("Restarting") == -1
```
Comment (kwypchlo) [inline playbooks/tasks/portals-check-nginx.yml:5]: you can use inspect templating

```bash
docker inspect -f '{{print .State.Restarting}}' nginx
```

**Review by kwypchlo — CHANGES_REQUESTED** [review]

**Comment by kwypchlo** [comment]
> This PR adds the check that the NGINX docker container is up and not restarting. It has a 15min timeout.
> 
> I added a note saying that the check is temporary, needed for next week's deploy. But it should be an instant pass in normal cases so there is no real harm in leaving it in.

I think it would be better to leave it there - we will need it in case we add any more certificates and we need to wait until they generate. Also we will need this check in new server setup ansible job.

File: playbooks/tasks/portals-check-nginx.yml (hunk 1 above)
Comment (firyx) [reply playbooks/tasks/portals-check-nginx.yml:5]: Or you can use docker filters:
`ansible.builtin.command: docker ps --filter name=nginx --filter status=running`
and check you got 2 lines - header and data line.

**Review by firyx — CHANGES_REQUESTED** [review]

File: playbooks/tasks/portals-check-nginx.yml (hunk 1 above)
Comment (kwypchlo) [reply playbooks/tasks/portals-check-nginx.yml:5]: > Or you can use docker filters:
> `ansible.builtin.command: docker ps --filter name=nginx --filter status=running`
> and check you got 2 lines - header and data line.
> 
> Anyway we should be aware of a possible issue while checking not `Restarting` or `running` status:
> Container might be in a `running` state like 100 ms before Nginx restarts and then move to `restarting` state in a loop.

is there a way to double check ? like have a condition where you need a success twice in a row with a delay of 1s for example

**Review by firyx — APPROVED** [review]

**Review by kwypchlo — APPROVED** [review]

**Pull request merged**